variables.tf

```diff
@@ -193,7 +193,7 @@ variable "node_pools" {
 }
 
 variable "cluster_logging_service" {
-  description = "The logging service to be used by the GKE cluster."
+  description = "The logging service that the cluster should write logs to. Available options include logging.googleapis.com (Legacy Stackdriver), logging.googleapis.com/kubernetes (Stackdriver Kubernetes Engine Logging), and none."
```

Comment:
Let's surround the acceptable string values with ". It's clearer to read in the docs that way.

Reply:
Added quotes